internal/browser.go

```diff
@@ -0,0 +1,38 @@
+package internal
+
+import (
+	"os/exec"
+	"runtime"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// OpenWebBrowser will try to open the default browser
+// on a the callers machine. It supports windows, darwin and windows.
+func OpenWebBrowser(url string) error {
```

Comment:
There's actually a nice little package we could use: https://github.com/pkg/browser

Reply:
I would rather open a ticket and implement this in a separate ticket as it affects other parts of spacectl too 🤷🏻